.bazelrc

```diff
@@ -67,6 +67,10 @@ build:darwin --host_platform=@rules_haskell//haskell/platforms:darwin_x86_64_nix
 # and GHC's gcc on Windows
 build:windows --crosstool_top=@rules_haskell_ghc_windows_amd64//:cc_toolchain
 
+# Bazel 1.0 disabled the bash test-runner on Windows. However, some of our
+# test-cases are implemented as bash scripts and rely on the bash test-runner.
+build:windows --noincompatible_windows_native_test_wrapper
```

Comment:
Not for this PR but could we work around this by having some wrapper binary that then calls bash internally?

Reply:
One way to address this is turn things like `client_server_test` into a macro that generates a script file and then calls `sh_test|binary` on that. Another is to use `cc_toolchain.link|compile` to generate a binary that calls the shell script. I think the former is the better approach. I've experimented with both and the latter caused some issues with the test-environment not matching what the test-cases expected.

Reply:
Sounds good, thanks for the explanation!